openradar/scripts/master.py

```diff
@@ -72,9 +72,16 @@ def master(**kwargs):
             tpl = 'Agg. task: {datetime} {timeframe}'
             logging.info(tpl.format(**aggregate_kwargs))
 
+            # Append nowcast aggregate subtask
```

Comment:
Here the nowcast aggregate task is submitted multiple times, for once for each product combination. It should be submitted only once for a given datetime.